selfdrive/car/tests/test_models.py

```diff
@@ -150,7 +150,8 @@ def test_car_interface(self):
       if i > 200:
         can_invalid_cnt += not CS.canValid
 
-    self.assertLess(can_invalid_cnt, 50)
+    print(can_invalid_cnt)
+    self.assertLess(can_invalid_cnt, 1)
```

Comment:
```suggestion
    self.assertLess(can_invalid_cnt, 50)
```

The one failure in CI was from using the last segment in a drive, msgs timed out. All expected